packages/integrations-hcl/strategies/nomad-pack/load_pack_directory.ts

```diff
@@ -42,13 +42,28 @@ export async function loadNomadPackIntegrationDirectory(
         `For Nomad Packs, the README file must be located at ./README.md, relative to the subdirectory of the Pack.`
     )
   }
-  const readmeContent: string = fs.readFileSync(readmeFile, 'utf8')
+
+  // Read the README
+  const rawReadmeContent: string = fs.readFileSync(readmeFile, 'utf8')
+
+  // If the first line is a H1, remove it. Keeping it will create a
+  // duplicate title in our content on the website.
+  const readmeLines = rawReadmeContent.split('\n')
+  const h1Regex = /^# .*/g
+  if (h1Regex.test(readmeLines[0])) {
```

Comment:
Suggestion, mostly for ourselves. Add a log statement to for this conditional action we're taking
... Something like: `console.log("Removing leading H1 from README content...")`